File: backend/Pms.Api/Models/Patient.cs
namespace Pms.Api.Models
{

    public class Patient
    {
        public int Id { get; set; }
        public string FullName { get; set; } = string.Empty;
        public string Address { get; set; } = string.Empty;
        public string? PhotoBase64 { get; set; }
        public ICollection<Appointment> Appointments { get; set; } = new List<Appointment>();
    }    
}

File: frontend/src/types/Patient.ts
import type { Appointment } from "./Appointment";

export interface Patient {
  id: number;
  fullName: string;
  address: string;
  photoBase64: string | null;
  appointments: Appointment[];
}

export interface PatientOverview {
  id: number;
  fullName: string;
}

export interface CreatePatientDto {
  fullName: string;
  address: string;
  photoBase64: string | null;
}